apis/v1alpha1/gatewayclass_types.go

```diff
@@ -65,8 +65,7 @@ type GatewayClassSpec struct {
 
 	// AllowedGatewayNamespaceSelector is a selector of namespaces that Gateways
 	// can use this GatewayClass from. This is a standard Kubernetes
-	// LabelSelector, a label query over a set of resources. The result of
-	// matchLabels and matchExpressions are ANDed. Controllers must not support
+	// LabelSelector. Controllers must not support
```

Comment:
This results in inconsistent line length, maybe just wrap at 80 chars. If you happen to use VSCode, [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) is awesome for automating this.